DNN Platform/Library/Services/Localization/Localization.cs

```diff
@@ -1245,13 +1243,7 @@ public static string GetStringUrl(string key, string resourceFileRoot)
         /// <returns>the string that is safe to use in a javascript function</returns>
         public static string GetSafeJSString(string unsafeString)
         {
-            var safeString = string.IsNullOrEmpty(unsafeString) ? "" : UnsafeJsRegex.Replace(unsafeString, "\\$1");
-			if (safeString.Length > 0)
-	        {
-				safeString = safeString.Replace("\r", string.Empty).Replace("\n", string.Empty);
-	        }
-
-			return safeString;
+            return HttpUtility.JavaScriptStringEncode(unsafeString);
```

Comment:
This will throw a `NullReferenceException` if passed a null, the existing code will return an empty string.